orb-ui/src/engine/diamond.rs

```diff
@@ -288,15 +289,17 @@ impl EventHandler for Runner<DIAMOND_RING_LED_COUNT, DIAMOND_CENTER_LED_COUNT> {
                     ring::alert::Alert::<DIAMOND_RING_LED_COUNT>::new(
                         Argb::DIAMOND_USER_QR_SCAN,
                         vec![0.0, 0.3, 0.3],
-                        false,
+                        None,
+                        true,
```

Comment:
Some food for thought: It would be more apparent what `true` means at the callsite, if you used an enum or an explicit type.